src/metabase/api/x_ray.clj

```diff
@@ -188,4 +203,15 @@
     "table" "table"
     "segment" "table"]])
 
+(def ^:private Settings
+  {:xray-max-cost costs/MaxCostBundles
+   :enable-xrays  s/Bool})
+
+(api/defendpoint PUT "/settings"
```

Comment:
I don't think we actually need this endpoint since everything happens via the existing setting PUT endpoint.